app/Modules/Form/form.view.php

```diff
@@ -13,12 +13,26 @@
 <?php } ?>
 
 <form action="<?= uri([FormController::class, 'store']) ?>" method="post">
-    <label for="name">Name:</label>
-    <input type="text" name="name" id="name">
+    <div>
+        <label for="name">Name:</label>
+        <input type="text" name="name" id="name" value="<?= $this->original('name') ?>">
```

Comment:
Two considerations to make on the long run:

- Should we add functions that allow us to skip `$this` everywhere?
- PHP as a templating engine isn't ideal, we should probably consider supporting another engine that has IDE support so that people can choose